docs/modules/ROOT/pages/concepts/RenovateBot/index.adoc

```diff
@@ -4,50 +4,12 @@ RenovateBot is a service that detects dependencies in your build pipeline and th
 
 RenovateBot monitors Tekton bundle references and makes sure that they're up to date. When new versions of the Tekton bundles are available, RenovateBot creates a pull request with the updated references. This gives you time to review, change, and incorporate the new bundles.
 
-== Configuration
-
-Configure RenovateBot through its `renovate.json` configuration file. The following is a sample `renovate.json` file: 
-
-Sample:
-
-----
-{
-
-
-    "$schema": "https://docs.renovatebot.com/renovate-schema.json",
-
-
-    "timezone": "America/New_York",
-
-
-    "schedule": ["before 10pm"],
-
-
-    "tekton": {
-
-
-    "fileMatch": ["\\.yaml$", "\\.yml$"],<1>
-
-
-    "pinDigests": true <2>
-
-
-    }
-
-
-}
-----
-
-Notice the following Tekton items in `renovate.json`: 
-
-<1> `link:https://docs.renovatebot.com/configuration-options/#filematch[fileMatch]`: An array type. RenovateBot parses each string in `fileMatch`, then extracts matching file names from a repository. In this sample, `fileMatch` directs RenovateBot to extract all `.yaml` and `.yml` files in your repository. 
-
-<2> `link:https://docs.renovatebot.com/configuration-options/#pindigests[pinDigests]`: A boolean type. When enabled, RenovateBot pins any Docker images or GitHub actions by means of their `SHA256` digest so that they're immutable.
-
-You can also update `renovate.json` with link:https://docs.renovatebot.com/configuration-options/#[many other options]. 
-
-== Additional information
-
-See an example in the xref:how-to-guides/configuring_renovatebot.adoc[Configuring RenovateBot] guide.
+== Authentication and authorization
+To create pull requests in target source code repositories Renovate bot has to have proper credentials.
 
+ * GitHub Application - Requires `Red Hat Konflux` GitHub App to be installed in the source code repository.
 
+ * GiLab. Create a link:https://docs.renovatebot.com/modules/platform/gitlab/#authentication[Personal Access Token] for the bot account.  Add PAT using link:https://redhat-appstudio.github.io/docs.appstudio.io/Documentation/main/how-to-guides/configuring-builds/proc_creating-secrets-for-your-builds/[
+Creating secrets for your builds]  The secret has to have such labels:
```

Comment:
```suggestion
Creating secrets for your builds].  The secret has to have such labels:
```

Reply:
When we say "such labels" are these just examples? or these are the labels that user should have?

In case these are examples, I believe we can update the sentence to: The secret should have labels, for example:

If these are something a secret must have: The secrets must have following labels:

Reply:
The secret must have the following labels:

+
* `appstudio.redhat.com/credentials: scm`
* `appstudio.redhat.com/scm.host: <your_domain_name> - For example,  appstudio.redhat.com/scm.host: gitlab.myhost.com